src/core/Akka/Serialization/ActorPathSerializer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Text;
+using Akka.Actor;
+
+namespace Akka.Serialization
+{
+    public class ActorPathSerializer : Serializer
```

Comment:
I don't think this is the best way to serialize an `ActorPath`.
For this to work, the `ActorPath` needs to be the root object, the message itself.
If the `ActorPath` is part of a larger message structure, this will fail since the serializer for that specific message type will kick in.

I think we should use the default serializer for this, this can be acheived by using substitution types in Json.NET.
This way, we can send both the ActorPath as the root message or as part of a bigger message..
See https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka/Actor/ActorRef.cs#L172
That would probably also require less code in the end, just those operator overloads.